apps/E2E/src/ButtonExperimental/pages/ButtonExperimentalPageObject.ts

```diff
@@ -10,14 +10,9 @@ class ButtonExperimentalPageObject extends BasePage {
   /******************************************************************/
   /**************** UI Element Interaction Methods ******************/
   /******************************************************************/
-  async didOnClickCallbackFire(): Promise<boolean> {
+  async didOnClickCallbackFire(errorMsg?: string): Promise<boolean> {
```

Comment:
This function looks/reads cleaner to me than `waitForOnClickCallbackToFire` in ButtonPageObject.ts. Could we replace `waitForOnClickCallbackToFire` in ButtonPageObject with this function (`didOnClickCallbackFire`)? Or at least make them both the same?